subjects/lunch_queue/README.md

```diff
@@ -2,27 +2,19 @@
 
 ### Instructions
 
-An API will have to be created to organize a queue of people.
+You will need to create an *API*, so that a program can organize a queue of people.
 
-Using the given code declare the following functions:
+The program requires the following functions. Add them as associated functions to the `Queue` structure:
 
-- `new` which will initialize the `Queue`
-- `add` which receives the person's information, so it can be added to the `Queue`
-- `invert_queue` which inverts the queue of persons
-- `rm`, which will remove the person who finished ordering their food.
-  The removal should respect a FIFO system (first in first out). This function should return a tuple wrapped in an `Option` with the information of the removed person (check the usage)
-- `search`, which returns a tuple with the information of a given person `id`
+- `new`: which will initialize the `Queue`.
+- `add`: which adds a person to the queue.
+- `invert_queue`: which reverses the queue.
+- `rm`: which removes the person who finished ordering their food. The removal should respect the FIFO method (first in first out). It should return the person's details.
+- `search`: which returns the details for a given person's `name`.
 
-You must also create a type called `Link`. This will be the connection of the structures `Queue` and `Person`.
-Do not forget that this will be a recursion type and it must point to `None` if there is no persons.
+You must also create a type named `Link`. This will be the connection of the structures `Queue` and `Person`. This will be a recursion type and it must point to `None` when the person is the last person in the queue.
```

Comment:
Is it in the last person or when there are no more people?

Reply:
Ah, I think you're right to point this out, as I guess we are talking about the `Queue` here, not `Person`.
The `Link` in `Queue` should point to a `Person` or `None` if there are no people.
The `Link` in `Persons` which we don't actually describe here should point to `None` when it is at the end of the *linked list* of people.

Reply:
I changed it here to make it a little more generic for `Queue` *and* `Person`: https://github.com/01-edu/public/pull/1109/commits/84226861cfa1f2116644ccf5479f68da1b6e560c

Reply:
Cool